functions/New-DbaAgentSchedule.ps1

```diff
@@ -534,7 +534,7 @@ function New-DbaAgentSchedule {
                             }
 
                             # Output the job schedule
-                            return $JobSchedule
+                            $JobSchedule
                         }
                     }
                 } # foreach object job
```

Comment:
Can you remove the comment on this line and push to the PR to get the test to run again, please.

Seems Pester hit an exception for the test on this command for some reason.

```Exception calling "LeaveTestGroup" with "2" argument(s): "TestGroups stack corrupted:  Expected name/hint of 'C:\github\dbatools\tests\New-DbaAgentSchedule.Tests.ps1','Script'.  Found 'New-DbaAgentSchedule Integration Tests', 
'Describe'."
At C:\Program Files\WindowsPowerShell\Modules\Pester\4.4.2\Pester.psm1:880 char:21
+ ...                    $pester.LeaveTestGroup($testScript.Path, 'Script')
+                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : ScriptMethodRuntimeException
 
Running C:\github\dbatools\tests\New-DbaAzAccessToken.Tests.ps1
Running C:\github\dbatools\tests\New-DbaClientAlias.Tests.ps1
```

Reply:
Interesting. Will make the change

Reply:
@wsmelton not sure if that comment you've meant (line 536), but the test failed again.

Reply:
Line 540